app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -49,6 +59,37 @@ Most teacher training courses in England are fee-paying. You'll need a Student v
 
 Some teacher training courses in England are salaried – this means you [work and earn a salary while you train](/funding-and-support/salaried-teacher-training). Currently, places on these courses are very limited. You'll need a Skilled Worker visa if you train on a salaried training course. Your Skilled Worker visa must be sponsored by the school employing you.
 
+<details id="details-non-uk-citizens-check-your-qualifications" class="expander-details expander-details__background-purple">
+  <summary class="expander-details__summary">
+    <div role="text">
+        <span class="expander-details__summary__header">
+          Refugees and asylum seekers:
+        </span>
+      <span class="expander-details__summary__title">
+        funding support
+      </span>
+</div>
+    <div class="expander-details__summary__show" aria-hidden="true">
+      <span class="expander-details__summary__icon expander-details__summary__icon__open" aria-hidden="true"></span>
+      <a class="link--underline" id="show-non-uk-citizens-check-your-qualifications" aria-hidden="true">Show</a>
+    </div>
+    <div class="expander-details__summary__hide" aria-hidden="true">
+      <span class="expander-details__summary__icon expander-details__summary__icon__close" aria-hidden="true"></span>
+      <a class="link--underline" id="hide-non-uk-citizens-check-your-qualifications" aria-hidden="true">Hide</a>
+    </div>
+  </summary>
+
+  <div class="expander-details__text">
+    <p>
+      </p>
+<p>If you <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees/non-uk-teachers/non-uk-qualifications">have refugee status you'll usually be eligible for financial support to help you train</a>.</p>
```

Comment:
This link isn't right - it's causing a build error

Should it be `/non-uk-teachers/fees-and-funding-for-non-uk-trainees`?